modules/oc-mirror-image-set-config-examples.adoc

```diff
@@ -9,47 +9,73 @@
 
 The following `ImageSetConfiguration` file examples show the configuration for various mirroring use cases.
 
+// Moved to first; unchanged
 [discrete]
-[id="oc-mirror-image-set-examples-helm_{context}"]
-== Use case: Including arbitrary images and helm charts
+[id="oc-mirror-image-set-examples-shortest-upgrade-path_{context}"]
+== Use case: Including the shortest {product-title} upgrade path
 
-The following `ImageSetConfiguration` file uses a registry storage backend and includes helm charts and an additional Red Hat Universal Base Image (UBI).
+The following `ImageSetConfiguration` file uses a local storage backend and includes all {product-title} versions along the shortest upgrade path from the minimum version of `4.11.37` to the maximum version of `4.12.15`.
 
 .Example `ImageSetConfiguration` file
 [source,yaml]
 ----
 apiVersion: mirror.openshift.io/v1alpha2
 kind: ImageSetConfiguration
-archiveSize: 4
 storageConfig:
- registry:
-   imageURL: example.com/mirror/oc-mirror-metadata
-   skipTLS: false
+  local:
+    path: /home/user/metadata
 mirror:
- platform:
-   architectures:
-     - "s390x"
-   channels:
-     - name: stable-4.13
- operators:
-   - catalog: registry.redhat.io/redhat/redhat-operator-index:v4.13
- helm:
-   repositories:
-     - name: redhat-helm-charts
-       url: https://raw.githubusercontent.com/redhat-developer/redhat-helm-charts/master
-       charts:
-         - name: ibm-mongodb-enterprise-helm
-           version: 0.2.0
- additionalImages:
-   - name: registry.redhat.io/ubi9/ubi:latest
+  platform:
+    channels:
+      - name: stable-4.10
```

Comment:
in the "stable-4.10" channel don't have the version 4.11.37. so the imageseconfig is not working . we could change to 'stable-4.12'

Reply:
Thanks, have updated to stable-4.12